design/use_cases/multi-story-building_use_case.md

```diff
@@ -0,0 +1,13 @@
+# Multi-story Building Navigation (2D+)
+As a Robot user I want my robot to be able to navigate stairways or elevators to move to another portion of the multi-story building so that it can do something useful
+
+## More details
+- Why is this needed?
+   - Example: a delivery robot in an office building 
+
+- What is the expected user interaction?
+   - The user should be able to specify a stairways and elevators on a map for a robot to use or not use
+
+- Are there any non-functional requirements? (build system, tools, performance, etc)
```

Comment:
Specifying this from yaml at the command line will be horrible. RVIZ or other GUI integration that allows users to click on maps and specify this would be much better.

Reply:
I'll update this one to state that we need both a GUI and a config file or API so that it can be done programmatically